llvm/utils/release/get-llvm-version.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+#===-- get-llvm-version.sh - Test the LLVM release candidates --------------===#
```

Comment:
The comment here is copy and paste wrong right?

Reply:
Fixed here: PR#101569.  I'll merge #101569 and then backport the change to the release branch.